src/main/cli/cli.c

```diff
@@ -6202,6 +6204,37 @@ static void cliDshotTelemetryInfo(const char *cmdName, char *cmdline)
         cliPrintLine("Dshot telemetry not enabled");
     }
 }
+
+static void cliDshotTelemetryData(const char *cmdName, char *cmdline)
+{
+    UNUSED(cmdName);
+    UNUSED(cmdline);
+
+    if (useDshotTelemetry) {
+        cliPrintLine("Motor    Type    RPM   TEMP    VCC   CURR  ST/EV   DBG1   DBG2   DBG3");
+        cliPrintLine("=====  ====== ====== ====== ====== ====== ====== ====== ====== ======");
+        for (uint8_t i = 0; i < getMotorCount(); i++) {
+            cliPrintLinef("%05d  %c%c%c%c%c %6d %02d.%02d %06d %06d %06d %06d %06d %06d",
```

Comment:
```suggestion
            cliPrintLinef("%5d   %c%c%c%c%c %6d  %2d.%2d %6d %6d %6d %6d %6d %6d",
```

Reply:
fixed

Reply:
Guess you forgot to remove the first 0 too. Now motors still show up as `00001` 😂

Reply:
fixed :)